cmd/create/oidcprovider/cmd.go

```diff
@@ -333,3 +339,12 @@ func validateOperatorRoles(awsClient aws.Client, cluster *cmv1.Cluster) ([]strin
 
 	return missingRoles, nil
 }
+
+func hasMode(mode string) bool {
```

Comment:
is there a better place to store a helper func like this as it is the same for operator roles and oidc providers?

Reply:
I don't think you'll need it, as you shouldn't be calling these commands unless there is a mode being passed to `rosa create cluster`